src/macros.rs

```diff
@@ -161,6 +161,43 @@ macro_rules! register_counter {
     }};
 }
 
+/// Create a [`Counter`] and registers to a custom registry.
+///
+/// # Examples
+///
+/// ```
+/// # #[macro_use] extern crate prometheus;
```

Comment:
I think you can turn this (and the rest of the examples too) into a `use prometheus::{register_counter_with_registry, opts}` and make it part of the example.

Reply:
Done